src/math/sat.rs

```diff
@@ -121,7 +121,7 @@ impl Intersector {
             let is_dupe = dedup_axes.iter().any(|ax2: &Unit<Vector3<f64>>| {
                 let d1 = (ax1.as_ref() - ax2.as_ref()).norm_squared();
                 let d2 = (ax1.as_ref() + ax2.as_ref()).norm_squared();
-                d1.min(d2) < f64::EPSILON
+                d1.min(d2) < std::f64::EPSILON
```

Comment:
do we need the `std::` prefix? 
https://blog.rust-lang.org/2020/04/23/Rust-1.43.0.html#library-changes

Reply:
On my laptop with v1.41 it didn't compile... I think it's not bad practice to support a bit outdated versions as well...

Reply:
:+1: